testsuite/functional-testsuite/src/test/ft-resources/testsuite-oss/security/security_ssl_certificates.t.js

```diff
@@ -0,0 +1,85 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2008-present Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+/**
+ * Tests SSL Certificate CRUD.
+ */
+StartTest(function(t) {
+
+  var sslCertificateListCQ = '>>nx-coreui-sslcertificate-list',
+      pemLoadCQ = '>>nx-coreui-sslcertificate-add-from-pem button[action=load]',
+      pemFieldCQ = '>>nx-coreui-sslcertificate-add-from-pem #pem',
+      addToTrustStoreButtonCQ = '>>nx-coreui-sslcertificate-details-form button[action=add][disabled=false]';
+
+  t.describe('SSL Certificates', function(t) {
+    t.it('An admin can load a certificate by pasting a PEM', function(t) {
+
+      t.chain(
+          t.openPageAsAdmin('admin/security/sslcertificates'),
+          {waitForCQVisible: sslCertificateListCQ},
```

Comment:
Looks to me like I always have to look in some other place to get the CQ so I like the previous version